tabcmd/commands/auth/session.py

```diff
@@ -274,17 +276,11 @@ def create_session(self, args, logger):
         self.logger = logger or log(__class__.__name__, self.logging_level)
 
         credentials = None
-        if args.password:
+        if args.password or args.password_file:
             self._end_session()
             # we don't save the password anywhere, so we pass it along directly
             credentials = self._create_new_credential(args.password, Session.PASSWORD_CRED_TYPE)
-        elif args.password_file:
-            self._end_session()
-            if args.username:
-                credentials = self._create_new_credential(args.password, Session.PASSWORD_CRED_TYPE)
-            else:
-                credentials = self._create_new_credential(args.password, Session.TOKEN_CRED_TYPE)
```

Comment:
Looks like `Session.TOKEN_CRED_TYPE` is no longer ever passed to `_create_new_credential`. Can we remove its `credential_type` argument and rename the method to `_create_new_password_credential`?

Reply:
yes - done